docker/app/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cd "${APP_DIR}"
+
+mkdir -p tmp/pids
+rm -f tmp/pids/server.pid
```

Comment:
Removes any leftover Rails server PID file to prevent startup conflicts when the container restarts.